### PR TITLE
Add mkumatag to milestone maintainers for test/images

### PR DESCRIPTION
**Pull request by mkumatag**

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml
Comment (justaugustus) [inline config/kubernetes/sig-release/teams.yaml:85]:
```suggestion
    - mkumatag # Testing: test/images
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@mkumatag -- One nit on the comment format.

I'm LGTM after that, but given that this is a SIG Testing directory, they should approve as well.
/hold
/assign @BenTheElder @spiffxp @stevekuznetsov

File: config/kubernetes/sig-release/teams.yaml
Comment (mkumatag) [reply config/kubernetes/sig-release/teams.yaml]: done!

**Comment by BenTheElder** [comment]
FWIW I don't mind @mkumatag, but I feel we've been fairly available for this sort of thing and I'm not sure I like the precedent here, there are already a TON of people that can do this, if we start adding every directory approver I'm not sure what the point of the list is..?

**Comment by justaugustus** [comment]
@BenTheElder -- we actually raised the meta topic on milestone-maintainers earlier today in the SIG Release meeting and @LappleApple has opened an issue to track --> https://github.com/kubernetes/sig-release/issues/1257

If you're happy with your current set of milestone maintainers, feel free to close this PR. :)

**Comment by mkumatag** [comment]
> FWIW I don't mind @mkumatag, but I feel we've been fairly available for this sort of thing and I'm not sure I like the precedent here, there are already a TON of people that can do this, if we start adding every directory approver I'm not sure what the point of the list is..?

Make sense, closing this PR.

**Pull request closed without merging**
